source/use-cases/sensitive-data-encryption.txt

```diff
@@ -13,26 +13,51 @@ Sensitive Data Encryption
 Introduction
 ------------
 
-Cras sed rutrum nisi, vitae tristique risus. Aliquam erat volutpat.
-Donec auctor nisl eget ornare accumsan. Pellentesque lectus tortor,
-consectetur molestie pellentesque ac, blandit non libero.
+If you store information in MongoDB such as confidential personal details, payment information, or proprietary data, you are probably interested in protecting your data from unauthorized access. In addition, the data you store may be subject to governance, privacy, and security compliance mandates. 
+
+MongoDB offers several methods that reduce your risk of a data breach resulting from unauthorized access whether accidental or by a malicious act including:
+
+* `Role-based access control <https://docs.mongodb.com/manual/core/authorization/>`_
```

Comment:
Thought this was relevant to talking about security features, but open to removing it or relocating it if it obscures the main purpose of the guide.

Reply:
It is definitely relevant - let's see if we can frame it in a way that shows a weakness or limitation of RBAC that is covered by CSFLE.

Reply:
One nit, I'd say "malicious actor"

Reply:
@terakilobyte 
Re: line 18, after reading the sentence a few times, I think it is not as crucial to mention the intentions of the data access. What do you think?
Here's a sample replacement sentence:
```
MongoDB offers several methods that protect your data from unauthorized access including:
```